tikv/deploy-tikv-using-ansible.md

```diff
@@ -0,0 +1,330 @@
+---
+title: Install and Deploy TiKV Using Ansible
+category: user guide
+---
+
+# Install and Deploy TiKV Using Ansible
+
+This guide describes how to install and deploy TiKV using Ansible. Ansible is an IT automation tool that can configure systems, deploy software, and orchestrate more advanced IT tasks such as continuous deployments or zero downtime rolling updates.
+
+[TiDB-Ansible](https://github.com/pingcap/tidb-ansible) is a TiDB cluster deployment tool developed by PingCAP, based on Ansible playbook. TiDB-Ansible enables you to quickly deploy a new TiKV cluster which includes PD, TiKV, and the cluster monitoring modules.
+
+## Prerequisites
+
+1. Several target machines that meet the following requirements:
+
+    - 4 or more machines
+
+        A standard TiKV cluster contains 6 machines. You can use 4 machines for test.
+
+    - CentOS 7.3 or later version of Linux operating system, x86_64 architecture (AMD64), ext4 filesystem
+
+        Use ext4 filesystem for your data disks. Mount ext4 filesystem with the `nodelalloc` mount option. See [Mount the data disk ext4 filesystem with options](../op-guide/ansible-deployment#mount-the-data-disk-ext4-filesystem-with-options).
+
+    - Network between machines
+
+        Turn off the firewalls and iptables when deploying and turn them on after the deployment.
+
+    - Same time and time zone for all machines with the NTP service on to synchronize the correct time.
+    
+        See [How to check whether the NTP service is normal](../op-guide/ansible-deployment#how-to-check-whether-the-ntp-service-is-normal).
+
+    - Create a normal `tidb` user account as the user who runs the service. 
+    
+        The `tidb` user can sudo to the root user without a password. See [How to configure SSH mutual trust and sudo without password](../op-guide/ansible-deployment#how-to-configure-ssh-mutual-trust-and-sudo-without-password).
+    
+    > **Note:** When you deploy TiKV using Ansible, use SSD disks for the data directory of TiKV and PD nodes.
+
+2. A Control Machine that meets the following requirements:
+
+    > **Note:** The Control Machine can be one of the target machines.
+    
+    - CentOS 7.3 or later version of Linux operating system (Python 2.7 involved by default)
+    - Access to the Internet
+    - Mutual trust of `ssh authorized_key` configured
```

Comment:
Can this also link to `../op-guide/ansible-deployment#how-to-configure-ssh-mutual-trust-and-sudo-without-password` ?